src/code/z_player_lib.c

```diff
@@ -621,7 +621,7 @@ PlayerItemAction func_80123810(PlayState* play) {
             play->interfaceCtx.unk_222 = 0;
             play->interfaceCtx.unk_224 = 0;
             Interface_SetHudVisibility(play->msgCtx.unk_120BC);
-            return PLAYER_IA_MINUS1;
+            return PLAYER_IA_HELD;
```

Comment:
You might also be interested in `func_80123810` just to help evaluate the `PLAYER_IA_HELD` name. This also ties into `play->unk_18794` which processes the exchange request, Currently:
```c
/**
 * Gives the player a request to use a C-Button to present an item to exchange
 * Return `exchangeItemAction` if an item is presented
 * Return `PLAYER_IA_NONE` if no option is offered yet
 * Return `PLAYER_IA_HELD` if the offer is declined or invalid
 */
func_80123810

```
Here's `func_80123810`
https://github.com/engineer124/mm/blob/b400822da32565533ff6d7a940a01c0aa373288a/src/code/z_player_lib.c#L616-L622
And `play->unk_18794`
https://github.com/engineer124/mm/blob/b400822da32565533ff6d7a940a01c0aa373288a/src/overlays/actors/ovl_player_actor/z_player.c#L10636
https://github.com/engineer124/mm/blob/b400822da32565533ff6d7a940a01c0aa373288a/src/overlays/actors/ovl_player_actor/z_player.c#L20923

And here's the main action for processing the exchange item:
https://github.com/engineer124/mm/blob/b400822da32565533ff6d7a940a01c0aa373288a/src/overlays/actors/ovl_player_actor/z_player.c#L17677

And some exchange item logic:
https://github.com/engineer124/mm/blob/b400822da32565533ff6d7a940a01c0aa373288a/src/overlays/actors/ovl_player_actor/z_player.c#L7412-L7441

Reply:
Discussed on discord,
decided to go with two enum values both equaling `-1` `PLAYER_IA_EXCH_CONTINUE` and `PLAYER_IA_HELD`.